src/main/java/de/ellpeck/rockbottom/net/chat/command/CommandGamemode.java

```diff
@@ -0,0 +1,50 @@
+package de.ellpeck.rockbottom.net.chat.command;
+
+import de.ellpeck.rockbottom.api.IGameInstance;
+import de.ellpeck.rockbottom.api.net.chat.Command;
+import de.ellpeck.rockbottom.api.net.chat.IChatLog;
+import de.ellpeck.rockbottom.api.net.chat.ICommandSender;
+import de.ellpeck.rockbottom.api.net.chat.component.ChatComponent;
+import de.ellpeck.rockbottom.api.net.chat.component.ChatComponentText;
+import de.ellpeck.rockbottom.api.util.reg.ResourceName;
+import de.ellpeck.rockbottom.world.entity.player.EntityPlayer;
+import de.ellpeck.rockbottom.api.entity.player.Gamemode;
+
+/**
+ * @author canitzp
+ */
+public class CommandGamemode extends Command {
+
+    public CommandGamemode() {
+        super(ResourceName.intern("gamemode"), "WIP", 3, "gamemode", "gm");
```

Comment:
You may as well put a proper description here (e.g. "Changes your gamemode. Usage `/gamemode <mode>`")